pyaedt/icepak.py

```diff
@@ -1464,7 +1475,7 @@ def create_assign_region2PCBComponent(self, scale_factor = 1.0, restore_padding_
 
         Returns
         -------
-
+        
```

Comment:
```suggestion
        tuple
            Tuple containing the ``(x, y, z)`` distances of the region.
```

Reply:
@PipKat, ``tuple`` type is a new one for you.  Lists are "mutable" meaning you can add things to them.  Tuples are not.  Returns from python are a tuple you return it with commas.  For example, here we have ``return dis_x, dis_y, dis_z``, which returns a tuple with three floats.